reduction/imaging_parameters.py

```diff
@@ -2938,7 +2938,7 @@
         "startmodel": "W43-MM2_B3_uid___A001_X1296_X11b_continuum_merged_12M_robust0_selfcal4_finaliter",
     },
     "W43-MM2_B6_12M_robust0": {
-        "threshold": "8.1mJy",  # "6mJy", #estimated noise: 2.7 mJy, from sio-only cube
+        "threshold": "6mJy",  # "6mJy", #estimated noise: 2.7 mJy, from sio-only cube
         # NOTE: 111/113 name ambiguous
```

Comment:
I don't understand the motivation; why not use the 3-sigma threshold?

Reply:
I think you can ignore this proposed change, after what we discussed in DR telecon. Yet it seems odd that this one source has a very specific 8.1 mJy number, whereas the respective param for the other sources is at 6 mJy. But I think Allison will change it to a sensible number for all fields.